text/0000-fabric-private-chaincode-lite.md

```diff
@@ -41,17 +42,37 @@ Such data includes secret cryptographic keys, which the chaincode uses to secure
     and also still mention all possible use-cases. 
     The distinction is done later in the architecture/design section
 -->
+
+***TODO*** document is missing a bit in the intro section about:
+- clear statement what is the purpose of the document
```

Comment:
hmm, this section seems to exactly specify the purpose of FPC Lite, so not 100% sure what is meant here. So they mean more on a meta level, i.e., what we ask from Fabric? (ie., acceptance of FPC lite as a "feature" and a corresponding repo)?